docs/xlpobjectheap.md

```diff
@@ -31,14 +31,10 @@ To find out the large page sizes available and the current setting, use the `-ve
 
 ## Syntax
 
-AIX&reg;, Linux&reg;, macOS&reg;, and Windows&trade;:
+AIX&reg;, Linux&reg;, macOS&reg;, Windows&trade; and Z/OS&reg;:
 
         -Xlp:objectheap:pagesize=<size>[,strict|warn]
 
-z/OS&reg;:
-
-        -Xlp:objectheap:pagesize=<size>[,strict|warn][,pageable|nonpageable]
```

Comment:
This still needs to be a valid option on z/OS, and shouldn't be removed from the docs.

Reply:
I note this option specification in incorrect until eclipse/openj9#7585 is merged, as pageable|nonpageable is not optional